src/scenes/Event/store.js

```diff
@@ -245,6 +229,7 @@ export default class EventStore {
     if (this.isWithdrawing) {
       await this.getDidWithdraw();
       await this.queryPendingWithdraw();
+      if (!this.event) return;
```

Comment:
this is already handled on L219?

Reply:
In event store it calls initEvent every new block, during this time if user goes to event history, `this.event` gets reset and become null, dapp would crash here.